packages/core/src/utils/TypeUtils.ts

```diff
@@ -70,3 +70,7 @@ export interface ApiGetWalletsResponse {
   data: ApiWallet[]
   count: number
 }
+
+export type ThemeMode = 'dark' | 'light'
+
+export type ThemeVariables = Record<string, string>
```

Comment:
If we know concrete values here, can we have better type completion by defining specific interface like
```
interface {
   '--w3m-font-family': string,
   // ...
}
```